guides/github.md

```diff
@@ -0,0 +1,130 @@
+# How to secure GitHub repositories
+
+It is recommended to turn on 2FA for most repositories, especially if building
+a package that is a dependency of multiple other packages.
```

Comment:
- 2FA can be easily circumvented with apps that understand TOTP, such as keepassxc
- 2FA apps are often installed on mobile devices, which are among the most insecure devices available

Reply:
That is true, but having 2FA adds an additional hurdle for attackers. I added a few more lines to include this comment